Hrm yeah but everything else is down.. and the initial connection test is failing 

```
Max attempts reached, failed to run 'test_connection netshoot-664cc4c98b-l6x74 100.1.208.30'!
[15:10:52.148] [shipyard]$ [cluster2] exit 1
make: *** [Makefile.inc:136: deploy] Error 1
[15:10:52.174] [shipyard]$ make --debug=b deploy
make: *** [Makefile.dapper:45: deploy] Error 2
``` 

Maybe the image is a no-go? I'm debugging manually now

--- a/scripts/shared/resources/nginx-demo.yaml
+++ b/scripts/shared/resources/nginx-demo.yaml
@@ -15,9 +15,9 @@ spec:
     spec:
       containers:
         - name: nginx-demo
-          image: quay.io/bitnami/nginx:latest
+          image: quay.io/testing-farm/nginx:latest
           ports:
-            - containerPort: 8080
+            - containerPort: 80
 ---
 apiVersion: v1
 kind: Service
@@ -30,7 +30,6 @@ spec:
   ports:
     - protocol: TCP
       port: 80
-      targetPort: 8080
   selector:
     app: nginx-demo
 ---

--- a/test/e2e/framework/deployments.go
+++ b/test/e2e/framework/deployments.go
@@ -106,7 +106,7 @@ func (f *Framework) NewNginxDeployment(cluster ClusterIndex) *corev1.PodList {
 					Containers: []corev1.Container{
 						{
 							Name:            "nginx-demo",
-							Image:           "quay.io/bitnami/nginx:latest",
+							Image:           "quay.io/testing-farm/nginx:latest",
 							ImagePullPolicy: corev1.PullAlways,
 							Ports: []corev1.ContainerPort{
 								{

